src/config/dynamic-theme-fixes.config

```
@@ -17100,6 +17100,64 @@ CSS
 
 ================================
 
+upwork.com
+/*---------------Home page fixes-------------------*/
+/* fix 1 Remove drop shadow of green button in right column */
+
+CSS
+.profile-completeness-nudges-tiles-alternative .carousel-wrapper .up-icon svg {
+ filter: drop-shadow(0 0 0px var(--white)) drop-shadow(0 0 0px var(--white)); 
+}
+/* ------fix 2 REMOVE WHITE GRADIENT HIGHLIGHTS------ */
+.up-skill-container .up-btn.up-btn-next::before { opacity: 0; }
+.up-skill-container .up-btn.up-btn-prev::before { opacity: 0; }
```

```suggestion
.up-skill-container .up-btn.up-btn-next::before,
.up-skill-container .up-btn.up-btn-prev::before {
    opacity: 0 !important;
}
```

Use the following style:
```
selector {
    CSS property: CSS value !important
}
```

---

I suppose it's resolved now right?